datasets/corona/corona

```diff
@@ -0,0 +1,111 @@
+{
+  "type": "dataset",
+  "id": "corona",
+  "title": "corona",
+  "status": "beschikbaar",
+  "description": "Corona datasets voor handhaving",
+  "version": "0.0.1",
+  "tables": [
+    {
+      "id": "handhaving",
+      "title": "handhaving",
+      "type": "table",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "required": [
+          "schema",
+          "id"
+        ],
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer",
+            "description": "Unieke aanduiding voor element"
+          },
+          "organisatie": {
+            "type": "string",
+            "description": "Organisatie"
+          },
+          "typeInterventie": {
+            "type": "string",
+            "description": "type van interventie bij handhaving"
+          },
+          "aantal": {
+            "type": "integer",
+            "description": "Aantal handhavingen voor deze week"
+          },
+          "week_nummer": {
+            "type": "integer",
+            "description": "Nummer van de week"
+          },
+          "jaar": {
+            "type": "integer",
+            "description": "jaartal in YYYY waarop aantallen betrekking hebben."
+          }
+        }
+      }
+    },
+    {
+      "id": "rivm",
```

Comment:
Ik denk dat we hem niet RIVM moeten noemen maar iets inhoudelijkers (want er zijn meer datasets van de RIVM.)
Iets als `gevallen` wellicht?

Reply:
Prima, pas het aan